docs/user/trusted-publishers/adding-a-publisher.md

```diff
@@ -99,8 +99,10 @@ each.
 
     For ActiveState, you must provide the name of the ActiveState project,
     the ActiveState organization that project belongs to, and the ActiveState user performing
+
     the publish action. Learn more about getting set up on the ActiveState Platform [here](https://docs.activestate.com/platform/start/PYPI).
```

Comment:
Weird, the change that was on line 103 here that fixes this URL looks to have gotten wiped out @th3coop

Reply:
There was a merge conflict.  I must have brought it back fixing that conflict